libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/MapPaddingAdjustor.java

```diff
@@ -1,16 +1,15 @@
 package com.mapbox.navigation.ui.map;
 
-import android.content.Context;
-import android.content.res.Resources;
-
-import com.mapbox.libnavigation.ui.R;
 import com.mapbox.mapboxsdk.maps.MapView;
 import com.mapbox.mapboxsdk.maps.MapboxMap;
 
+import static com.mapbox.navigation.ui.map.MapPaddingCalculator.calculateDefaultPadding;
+
 class MapPaddingAdjustor {
 
-  private static final int BOTTOMSHEET_PADDING_MULTIPLIER = 4;
-  private static final int WAYNAME_PADDING_MULTIPLIER = 2;
+  public static final int BOTTOMSHEET_PADDING_MULTIPLIER_PORTRAIT = 4;
```

Comment:
shouldn't it be moved to `MapPaddingCalculator` and make internal?

Reply:
I thought about that but the calculator is just a function it seemed like the constants belonged to the class since the behavior is encapsulated in the class. I didn't see any harm in them being public.

Reply:
Isn't a bit strange if the holding class is `internal` 🤔?